devel/patchelf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                patchelf
+version             0.10.0
+categories          devel
+platforms           darwin
+license             GPL-3
```

Comment:
upstream states "either version 3 of the License, or (at your option) any later version." So please list the license as `GPL-3+`